frontend/rolecall/src/app/cast/cast-drag-and-drop.component.scss

```diff
@@ -41,26 +42,57 @@ $cellHeight: 4rem;
   margin: 0;
   overflow-x: hidden;
   overflow-y: hidden;
-  padding: 1em;
+  padding: 1.2em;
   text-overflow: ellipsis;
   white-space: nowrap;
   width: $cellWidth;
   &ColHeader {
+    padding-top: 1.45rem;
     color: $gray;
     font-size: large;
     font-weight: bold;
     vertical-align: middle;
   }
   &RowHeader {
+    padding-top: 1.45rem;
+    padding-left: 2rem;
     color: $gray;
     font-size: large;
     font-weight: bold;
     vertical-align: middle;
   }
   &BothHeader {
+    padding-top: 1.4rem;
+    padding-left: 1rem;
+    color: $black;
+    font-size: 1.4em;
+    font-weight: bold;
+    vertical-align: middle;
+    width: $bothWidth;
   }
 }
 
+.dancer-count-buttons-container {
+  background: $offWhite;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 1.5rem;
+}
+.dancer-count-button {
+  background: $offWhite;
+  height: 2rem;
+  width: 1rem;
+  min-height: 5px!important;
```

Comment:
Can we avoid !important here? Usually we consider it an anti-pattern and reserve it for library-level stuff.

Reply:
Done. It was a debug class that I forgot to remove.

Reply:
Done.